src/runtime/file_util.cc

```diff
@@ -142,5 +142,9 @@ void LoadMetaDataFromFile(
   fs.close();
 }
 
+void RemoveFile(const std::string& file_name) {
+  std::remove(file_name.c_str());
```

Comment:
This does not seems to be right.

Reply:
I think it is fine to not have remove functionality in C++ and request per RPC server type registration